integration/tests/cook/test_client.py

```diff
@@ -56,3 +59,37 @@ def test_kill(self):
         self.client.kill(uuid)
         job = self.client.query(uuid)
         self.assertEqual(job.status, JobStatus.COMPLETED)
+
+    def test_container_submit(self):
```

Comment:
We probably want:

```suggestion
    @unittest.skipUnless(util.docker_tests_enabled(), "Requires a test docker image")
    def test_container_submit(self):
```

Reply:
I'm a little confused here, cause I did put this decorator (otherwise the integration tests would fail) and it doesn't seem to be showing up on the Github UI here. But if you look [here](https://github.com/hdrodz97/Cook/blob/client-fixes/integration/tests/cook/test_client.py#L65-L77), the decorator is in fact showing up. Hopefully this is a snag on the Github UI's end and it'll clear out after the next round of commits.

Reply:
Okay it's showing up now, that was weird.